File: learn2.js
function ask (a,b){

    if (a == "B001"){
        total = 15000*b;
        pajak = total *0.11;
        console.log("Total Penjualan = "+total);
        console.log("Pajak = " +pajak);
        grand = total + pajak;
        console.log("Grand = " +grand);
    }
    else if (a == "B002"){
        total = 18000*b;
        pajak = total *0.11;
        console.log("Total Penjualan = "+total);
        console.log("Pajak = " +pajak);
        grand = total + pajak;
        console.log("Grand = " +grand);
    }
    else if (a == "B003"){
        total = 24000*b;
        pajak = total *0.11;
        console.log("Total Penjualan = "+total);
        console.log("Pajak = " +pajak);
        grand = total + pajak;
        console.log("Grand = " +grand);
    }
        
    else if(a == "B004"){
        total = 18000*b;
        pajak = total *0.11;
        console.log("Total Penjualan = "+total);
        console.log("Pajak = " +pajak);
        grand = total + pajak;
        console.log("Grand = " +grand);
    }
}

ask("B004",2);

File: learn2.py
def ask ():
    a = input("Masukkan kode barang : ")
    b = int(input("Masukkan Jumlah : "))

    if a == "B001" :
        total = 15000*b
        pajak = total *0.11
        print(f"Total Penjualan = {total}")
        print(f"Pajak = {pajak}")
        grand = total + pajak
        print(f"Grand = {grand}")
    elif a == "B002" :
        total = 18000*b
        pajak = total *0.11
        print(f"Total Penjualan = {total}")
        print(f"Pajak = {pajak}")
        grand = total + pajak
        print(f"Grand = {grand}")
    elif a == "B003" :
        total = 24000*b
        pajak = total *0.11
        print(f"Total Penjualan = {total}")
        print(f"Pajak = {pajak}")
        grand = total + pajak
        print(f"Grand = {grand}")
        
    elif a == "B004" :
        total = 18000*b
        pajak = total *0.11
        print(f"Total Penjualan = {total}")
        print(f"Pajak = {pajak}")
        grand = total + pajak
        print(f"Grand = {grand}")

ask()